website/docs/others/contributing.md

```diff
@@ -1,3 +1,103 @@
 ---
 sidebar_position: 2
 ---
+
+# Contributing
+
+## Welcome
+
+Welcome to CONTRIBUTING zone. If you are reading this, you probably want to contribute to Open Source projects. That's great! Contributing to an open source project is a great opportunity to learn, sharpen your skills and help others. Luckily, Jest Preview is a free and open source project and we always welcome new contributors. Its mission is to help front end developers' lives easier and more enjoyable, also improve the standard of front end applications by encouraging them to write more high-quality tests, by providing an ability to view the actual UI in Jest in an external browser like Chrome.
+
+Jest Preview aims to be a community-driven project. So we hope to see your contributions to make Jest Preview a favorite library for front end developers when it comes to testing.
+
+## What can I contribute?
+
+We appreciated any help. There is not a thing as a small contribution. If you see a typo, send us a PR. If you experience a bug, please open an issue. If you have a suggestion, let us know. Below are some ways you can contribute to Jest Preview:
```

Comment:
Actually, when building the docs website, we will copy the content from `CONTRIBUTING.md` to `website/docs/others/contributing.md`.
You can see the source code [here](https://github.com/nvh95/jest-preview/blob/38ed47d6e456984fa2c0d418d9afbf63d963d3c0/website/scripts/contributing.js) and the docs website that already has contributing guideline:
https://www.jest-preview.com/docs/others/contributing

(You can see that contributing is duplicated in https://deploy-preview-196--jest-preview-library.netlify.app/docs/others/contributing)

I guess you want to reverse the changes in this file.
If you want to update contributing guideline, please make the change directly in `CONTRIBUTING.md`

Reply:
Done